app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,361 @@
+</Text>
+
+<Text>
+If you want to create new functionalities for Oskari, it might be a good idea to gather together all the ideas that you have and hand them over to the development team. Thus they can see which are top of your priorities now and what kind of things to expect next - and also suggest to you which of your ideas could be fulfilled at the same time quite easily. By coming up with multiple ideas at once they can also take the upcoming ideas into consideration when developing the first new functionalities.
+</Text>
+
+<Text>
+With new functionalities, one should remember that they must be maintained. When you create new functionalities to your Oskari service, remember to inform NLS both beforehand and after the development has taken place. By contacting the NLS FI&apos;s Oskari technical team before you start the development, your developers can get best practices for the development process and further ideas on how to create the functionalities that you want. 
+</Text>
+
+<Text>
+When the new functionalities are in use, one should publish the code in a GitHub repository and inform the Oskari technical team - this helps them to take into account the dependencies of the new functionality when they are making new versions of Oskari. There might be dependencies, libraries or functionalities in the Oskari core that are affect the new functionality, and if you don’t inform the Oskari technical team about what you are doing or what you have done, it is possible that something gets broken when it&#39;s the time to update the Oskari instance next time.
+</Text>
+
+<Text>
+The changes into Oskari should never be done into the core of Oskari but as a separate whole. This makes the updating of the Oskari core much easier in the future.
+</Text>
+
+<Text>
+And to make sure that Oskari is a living, developing open source project for years to come for your organisation and everybody else, <b>it is highly recommended that you also contribute to the main Oskari GitHub branch and inform other Oskari users about what you are doing or have done.</b> This way other Oskari users can both enjoy the new functionalities and possibilities of Oskari but also contribute to the further development of the software. Oskari is open source at heart and if you create something Oskari-related, please publish it as open source.
```

Comment:
Github branch -> repositories. Informing about new functionalities are then going to come in Oskari release notes when the contributed functionality has been merged and released in an Oskari version.